iceaxe/schemas/db_serializer.py

```diff
@@ -172,8 +181,37 @@ async def get_constraints(self, session: DBConnection, table_name: str):
                 )
                 target_columns = {row["column_name"] for row in target_columns_result}
 
+                # Map PostgreSQL action codes to action strings
+                action_map = {
+                    "a": "NO ACTION",
+                    "r": "RESTRICT",
+                    "c": "CASCADE",
+                    "n": "SET NULL",
+                    "d": "SET DEFAULT",
+                }
+
+                on_update = action_map.get(
+                    row["confupdtype"].decode()
+                    if isinstance(row["confupdtype"], bytes)
+                    else row["confupdtype"],
```

Comment:
We're now using this same conversion logic in a few places since it's not well defined whether we get a bytes array or a string back from these query calls. Consider adding a small helper `_unwrap_db_as_str()` or similar that handles this case.